data-model/build.gradle.kts

```diff
@@ -16,6 +16,20 @@ dependencies {
     api("org.apache.commons:commons-compress:1.21") {
       because("Multiple vulnerabilities in avro-declared version")
     }
+    api("com.fasterxml.jackson.core:jackson-core:2.13.2") {
```

Comment:
@laxman-traceable jackson-core doesn't have `2.13.2.1` so upgraded to `2.13.2`

Reply:
I checked - it's avro, so just fixing databind should be correct.

Reply:
yes, its Avro, and I had tried the latest Avro version.

``` 
  ✗ Denial of Service (DoS) [High Severity][https://snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-2421244] in com.fasterxml.jackson.core:jackson-databind@2.12.5
    introduced by org.apache.avro:avro@1.11.0 > com.fasterxml.jackson.core:jackson-databind@2.12.5
  This issue was fixed in versions: 2.13.2.1
  ```
  Yes, just data-bind should be fine as per deps, updating the PR.
  ```
+--- org.apache.avro:avro:1.10.2
|    +--- com.fasterxml.jackson.core:jackson-core:2.12.2 -> 2.13.2
|    |    \--- com.fasterxml.jackson:jackson-bom:2.13.2 -> 2.13.2.1
|    |         +--- com.fasterxml.jackson.core:jackson-annotations:2.13.2 (c)
|    |         +--- com.fasterxml.jackson.core:jackson-core:2.13.2 (c)
|    |         \--- com.fasterxml.jackson.core:jackson-databind:2.13.2.1 (c)
|    +--- com.fasterxml.jackson.core:jackson-databind:2.12.2 -> 2.13.2.1
|    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.13.2
|    |    |    \--- com.fasterxml.jackson:jackson-bom:2.13.2 -> 2.13.2.1 (*)
|    |    +--- com.fasterxml.jackson.core:jackson-core:2.13.2 (*)
|    |    \--- com.fasterxml.jackson:jackson-bom:2.13.2.1 (*)
|    +--- org.apache.commons:commons-compress:1.20 -> 1.21
|    \--- org.slf4j:slf4j-api:1.7.30
```